src/Modal/index.js

```diff
@@ -188,9 +194,9 @@ ModalWrapperContent.defaultProps = {
   theme: defaultTokens,
 };
 
-const CloseElement = ({ onClose }: CloseElementType) => (
+const CloseElement = ({ onClose, closable }: CloseElementType) => (
   <CloseContainer>
-    <ButtonLink onClick={onClose} size="normal" icon={<Close />} transparent />
+    {closable && <ButtonLink onClick={onClose} size="normal" icon={<Close />} transparent />}
```

Comment:
please this condition to `319` line before render the whole `CloseElement`

Reply:
You can't the `CloseContainer` is used as background for the mobile header (when scrolled).

Reply:
OK, but it's little bit confusing.

Reply:
We will see how it will looks like after another update :)